File: Programmers/2021_WeeklyChallenge/2W.py
def returnGrade(avg):
    if avg >= 90:
        return 'A'
    elif avg >= 80 and avg < 90:
        return 'B'
    elif avg >= 70 and avg < 80:
        return 'C'
    elif avg >= 50 and avg < 70:
        return 'D'
    else:
        return 'F'

def solution(scores):
	answer = ''
	
	for i in range(len(scores)):
		data = []
		score = 0
		for j in range(len(scores[0])):
			data.append(scores[j][i])
		if data[i] is max(data):
			if data.count(max(data)) == 1:
				data.remove(max(data))
		elif data[i] is min(data):
			if data.count(min(data)) == 1:
				data.remove(min(data))
		score = sum(data) / len(data)
		answer += returnGrade(score)
	return answer

File: Programmers/2021_WeeklyChallenge/2W.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string returnGrade(int avg){
    if(avg >= 90) return "A";
    else if (avg >= 80 && avg < 90) return "B";
    else if (avg >= 70 && avg < 80) return "C";
    else if (avg >= 50 && avg < 70) return "D";
    else return "F";
}

string solution(vector<vector <int>> scores){
    string answer = "";
    vector<vector<int>> new_scores(scores.size(), vector<int> (scores.size()));
    int max, min, 
    int score = 0;
    for(int i = 0; i < scores.size(); i++){
        for(int j = 0; j < scores[i].size(); j++) new_scores[i][j] = scores[j][i];
    }
    scores = new_scores;
    for(int i = 0; i < scores.size(); i++){
        min = *min_element(scores[i].begin(), scores[i].end());
        
    }
    return answer;
}